frontend/src/components/details-card/DetailsCard.tsx

```diff
@@ -1,26 +1,39 @@
+import Label from "components/label/Label";
+import Value from "components/value/Value";
 import React, { FunctionComponent, ReactElement } from "react";
 import Card from "../card/Card";
 import classes from "./DetailsCard.module.scss";
 
-type DetailsCardProps = {
-  header?: ReactElement;
-  footer?: ReactElement;
+export type column = Array<{
+  label: ReactElement | string;
+  value: ReactElement | string;
+}>;
+
+export type DetailsCardProps = {
+  columns: column[];
 };
```

Comment:
I belive a more readable way to write this could be:

```tsx
export type DetailsCardRow = {
  label: ReactElement | string;
  value: ReactElement | string;
};

export type DetailsCardColumn = DetailsCardRow[];

export type DetailsCardProps = {
  columns: DetailsCardColumn[];
};
```